lib/elixir_analyzer.ex

```diff
@@ -29,18 +29,14 @@ defmodule ElixirAnalyzer do
 
   * `exercise` is which exercise is submitted to determine proper analysis
 
-  * `path` is the path (ending with a '/') to the submitted solution
+  * `input_path` is the path to the submitted solution
+
+  * `output_path` is the path to the output folder
 
   * `opts` is a Keyword List of options, see **options**
 
   ## Options
 
-  * `:exercise` - name of the exercise, defaults to the `exercise` parameter
-
-  * `:path` - path to the submitted solution, defaults to the `path` parameter
-
-  * `:output_path` - path to write file output, defaults to the `path` parameter
```

Comment:
I did not understand these options here, all they do is overwrite the required arguments. Those options were passed from the CLI module. I got rid of them.